### ISSUE TITLE
"gather" environment not recognized

### ISSUE THREAD

**Issue by iXeRay** [sylvainhalle/textidote#229]
Hi,

I have started using textidote recently and I came across what looks like an unwanted behavior.
The "gather" environment doesn't seem to be recognized contrary to "align" or "equation" for instance.

As a result, the \begin{gather} line is flaged as a spelling mistake, and multiple warnings are raised inside the environment.

**Comment by sylvainhalle** [comment]
That's right, `gather` should be added to the environment names on these lines:

https://github.com/sylvainhalle/textidote/blob/eacdb0f56acbe6edcc6ae8f3223aab895f51f4a5/Source/Core/src/ca/uqac/lif/textidote/cleaning/latex/LatexCleaner.java#L234
https://github.com/sylvainhalle/textidote/blob/eacdb0f56acbe6edcc6ae8f3223aab895f51f4a5/Source/Core/src/ca/uqac/lif/textidote/cleaning/latex/LatexCleaner.java#L234
